src/components/Album.jsx

```diff
@@ -1,17 +1,19 @@
-import { PropTypes } from 'prop-types';
-import { AlbumName } from './AlbumName';
-import { CoverImage } from './CoverImage';
+import { PropTypes } from "prop-types";
+import { AlbumName } from "./AlbumName";
+import { ArtistName } from "./ArtistName";
+import { CoverImage } from "./CoverImage";
 
 export const Album = ({ album }) => {
   // TODO: remove this later on
   // console.log(album);
 
-  const image = album.images.find(image => image.height === 640);
+  const image = album.images.find((image) => image.height === 640);
 
   return (
     <div className="album-container">
       <CoverImage img={image.url} alt={album.name} />
       {<AlbumName albumName={album.name} uri={album.uri} />}
+      {<ArtistName className="artist-name" artists={album.artists} />}
```

Comment:
we don't need to wrap this component in curly braces. We can also remove them for line 15.

Reply:
I see! For some reason I thought that was a React thing. Will remove them for bort components!